File: github/modul 4/Bahasa py/PRAK404-241081732010-RabiahRiskaAmaliah.py
while True:
    print("Pilih Program ")
    print("1. Penjumlahan ")
    print("2. Pengurangan ")
    print("3. Perkalian ")
    print("4. Pembagian ")
    print("5. Exit")

    i= int (input("Masukan pilihan: "))

    if i == 5:
       print("Terimakasih, telah menggunakan kalkulator Rabiah Riska Amaliah \n")
       break 
    elif i <= 0 or i > 5:
        print("Input anda salah, silahkan coba lagi")
        continue
    else:
        a = float (input("Masukan nilai pertama:"))
        b = float (input("Masukan nilai kedua:"))
        if i == 1:
             print("Hasil penjumlahan antara {:.2f} dan {:.2f} adalah {:.2f}".format(a, b, a + b))
        elif i == 2:
            print("Hasil pengurangan antara {:.2f} dan {:.2f} adalah {:.2f}".format(a, b, a - b))
        elif i ==3:
            print("Hasil perkalian antara {:.2f} dan {:.2f} adalah {:.2f}".format(a, b, a * b))
        elif i==4:
            print("Hasil pembagian antara {:.2f} dan {:.2f} adalah {:.2f}".format(a, b, a / b))

   

File: github/modul 4/Bahasa C/PRAK404-241081732010-RabiahRiskaAmaliah.c
#include <stdio.h>

int main (){
    int i;
    float a, b, hasil;
    
while (1){
    printf("Pilih Program \n");
    printf("1. Penjumlahan \n");
    printf("2. Pengurangan \n");
    printf("3. Perkalian \n");
    printf("4. Pembagian \n");
    printf("5. Exit \n"); 
    printf("Masukan pilihan:"); 
    scanf("%d", &i);

    if (i==5) {printf("Terimakasih, telah menggunakan kalkulator Rabiah Riska Amaliah\n"); break;} 
    else if (i <= 0 || i > 5) {printf("Input anda salah, silahkan coba lagi\n");continue;}
    else {
    printf("Masukan nilai pertama:");
    scanf("%f", &a);
    printf("Masukan nilai kedua:");
    scanf("%f", &b);}
    if (i == 1) {printf("Hasil penjumlahan antara %.2f dengann %.2f adalah %.2f\n", a, b, a+b);}
    else if (i == 2) {printf("Hasil pengurangan antara %.2f dengann %.2f adalah %.2f\n", a, b, a-b);}
    else if (i == 3) {printf("Hasil perkalian antara %.2f dengann %.2f adalah %.2f\n", a, b, a*b);}
    else if (i == 4) {printf("Hasil pembagian antara %.2f dengann %.2f adalah %.2f\n", a, b, a/b);}
}
    return 0;
}